### DISCUSSION TITLE
PFD and ND

### DISCUSSION THREAD

**Discussion by BEHAD** [flybywiresim/aircraft#4002] [Help]
Hallo,
nach dem letzten MSFS Update Fehler bei den Displays im A32NX

In der A32NX Stable Version bleiben die PFD und ND Displays dunkel!
In der A32NX nur die ND Displays!

Hat jemand eine Lösung dafür???

Vielen Dank

**Comment by MapleNAV-TCASFAULT** [comment]
Verwenden Sie bei Bedarf Google Translate

The latest a32nx stable version v0.5.4 should work, if it still doesn't work make sure that you don't have any conflicting mods. So try clearing out your community folder.

**Comment by BEHAD** [comment]
Sorry, meine Frage war nicht ganz korrekt.
Hier die Überarbeitung:

Hallo,
nach dem letzten MSFS Update Fehler bei den Displays im A32NX

In der A32NX Stable Version bleiben die PFD und ND Displays dunkel!
In der A32NX Development Version nur die ND Displays dunkel!!

Hat jemand eine Lösung dafür???

Vielen Dank
